rpm/update-spec-provides.sh

```diff
@@ -4,12 +4,27 @@
 # packaging, via the `propose-downstream` packit action.
 # The goimports don't need to be present upstream.
 
-set -eo pipefail
+set -eox pipefail
 
 PACKAGE=skopeo
 # script is run from git root directory
 SPEC_FILE=rpm/$PACKAGE.spec
 
+# Needed for golist to work
+# This is run in the packit sandbox environment for downstream tasks and
+# doesn't affect upstream repo and CI.
+GOPATHDIR=~/go/src/github.com/containers/
+mkdir -p $GOPATHDIR
+ln -sf $(pwd) $GOPATHDIR/.
```

Comment:
I can’t see `golist`, nor the Go standard library, ever referring to `GOPATHDIR`. (Comparing https://pagure.io/go-rpm-macros/blob/master/f/rpm/macros.d/macros.go-rpm#_187 .)